corehq/blobs/tests/test_s3db.py

```diff
@@ -144,6 +145,31 @@ def test_delete_bucket(self):
         with self.assertRaises(mod.NotFound):
             self.db.get(info.identifier, bucket=bucket)
 
+    def test_delete_identifier_in_default_bucket(self):
+        info = self.db.put(StringIO(b"content"))
+        self.assertTrue(self.db.delete(info.identifier), 'delete failed')
+        with self.assertRaises(mod.NotFound):
+            self.db.get(info.identifier)
+
+    def test_delete_no_args(self):
+        info = self.db.put(StringIO(b"content"))
+        with self.assertRaises(ArgumentError):
+            self.db.delete()
+        # blobs in default bucket should not be deleted
+        with self.db.get(info.identifier) as fh:
+            self.assertEqual(fh.read(), b"content")
+        self.assertTrue(self.db.delete(bucket=mod.DEFAULT_BUCKET))
+
+    def test_prevent_delete_bucket_by_mistake(self):
+        info = self.db.put(StringIO(b"content"))
+        id_mistake = None
+        with self.assertRaises(ArgumentError):
+            self.db.delete(id_mistake, mod.DEFAULT_BUCKET)
+        # blobs in default bucket should not be deleted
+        with self.db.get(info.identifier) as fh:
+            self.assertEqual(fh.read(), b"content")
+        self.assertTrue(self.db.delete(bucket=mod.DEFAULT_BUCKET))
```

Comment:
These tests seem identical to the fsdb ones. Is it not possible to reuse them somehow?